bower.json

```diff
@@ -12,6 +12,7 @@
     "o-loading": "^3.0.0",
     "o-typography": "^5",
     "o-viewport": "^3.0.1",
-    "o-normalise": "^1.6.2"
+    "o-normalise": "^1.6.2",
+    "o-buttons": "^5.16.6"
```

Comment:
Adding a new dependency technically makes this a breaking change and requires the release to be a new major version. This is because a consumer of o-video could be using o-buttons already and we would like to avoid a conflict being returned to the consumer (if they were using the origami build service, it would cause their website to break if a conflict was served 😔 )

Reply:
Great to know! I've managed to remove the need for o-buttons all together 👍

Reply:
If that's the case, you need to remove `o-buttons` from `bower.json` as well, otherwise it is still a major version change.